README.md

```diff
@@ -248,6 +261,12 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [H2O](http://0xdata.com/) - Analytics engine for statistics over big data.
 * [Weka](http://www.cs.waikato.ac.nz/ml/weka/) - Collection of algorithms for data mining tasks ranging from pre-processing to visualization.
 
+## Mail
+
+*Libraries that simplify sending and receiving emails.*
+
+* [Jodd EMail](http://jodd.org/doc/email.html) - Java mail, fluent and simplified.
```

Comment:
I don't understand the benefit as compared to the simple mail client of Apache Commons.